src/app/core/shared/media-viewer-item.model.ts

```diff
@@ -14,6 +14,11 @@ export class MediaViewerItem {
    */
   format: string;
 
+  /**
+   * Incoming Bitsream format mime type
+   */
+  minetype: string;
```

Comment:
@yingjin : There's a spelling error here.  You've accidentally spelled it `minetype` instead of `mimetype`.  Could you quickly correct this?